rules/linux/lnx_security_tools_disabling.yml

```diff
@@ -2,11 +2,11 @@ title: Disabling Security Tools
 id: e3a8a052-111f-4606-9aee-f28ebeb76776
 status: experimental
 description: Detects disabling security tools
-author: Ömer Günal
+author: Ömer Günal, Alejandro Ortuno
 date: 2020/06/17
 references:
-    - https://attack.mitre.org/techniques/T1089/
     - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1089/T1089.md
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1562.004/T1562.004.md
 logsource:
     product: linux
```

Comment:
I believe that it makes sense to add here `process_creation` category or use the old auditd way. At least one approach should be declared

Reply:
So I see your point but would not this conflict with for example the `CarbonBlack` as I am missing how this would relate if we specify the category to be the `process_creation`.

Reading the documentation: https://github.com/Neo23x0/sigma/wiki/Rule-Creation-Guide#log-source

Would not be the carbonblack part of a different `product`?

Reply:
Hi the rule is under `rules/linux/`  so in my point of view it would make more sense:
  1- to see only linux logs in this directory
  2 - logsource: ` product: linux service: auditd`   will be more suitable like https://github.com/Neo23x0/sigma/blob/c17ca6d5fe350c7c25f782d6016c65132901190c/rules/linux/auditd/lnx_data_compressed.yml
 3- finally, for linux logs, it could be more interesting to use sub categories like windows (see:https://github.com/Neo23x0/sigma/tree/master/rules/windows) and `process_creation` will refer to `auditd execve` (create an issue for sigma maintainers ?)

Reply:
@alejandroortuno Carbon Black had process execution log as well.  `eventType` "CREATE_PROCESS", to be more precise:

```
{
  "_source": {
    "longDescription": "The application \"<share><link hash=\"1f41e79753e4a75dfee7a9bd309a40ed21e7b69523e8b152e3df4ad71455d8f2\">/bin/zsh</link></share>\" invoked the application \"<share><link hash=\"b0ddc6174508b39afd43810089d8e00ba423a714464a4aa9fcbb40480d40bec0\">/usr/local/Cellar/bash/5.0.18/bin/bash</link></share>\". ",
    "alertScore": 0,
    "eventTime": 1603748280740,
    "processDetails": {
      "targetCommandLine": "bash -c echo hello, @alejandroortuno",
      "targetName": "bash",
      "interpreterName": null,
      "privatePid": "18751-132481386922480000-0",
      "milisSinceProcessStart": "depersonalized",
      "parentPid": "depersonalized",
      "userName": "depersonalized",
      "parentPrivatePid": "depersonalized",
      "parentCommandLine": "depersonalized",
      "parentName": "login",
      "targetPrivatePid": "depersonalized",
      "processId": "depersonalized",
      "interpreterHash": null,
      "fullUserName": "depersonalized",
      "name": "zsh",
      "targetPid": "depersonalized",
      "commandLine": "-zsh"
    },
    "tag": "depersonalized",
    "alertCategory": null,
    "netFlow": {
      "peerIpV4Address": null,
      "sourcePort": null,
      "sourceAddress": "depersonalized",
      "destPort": null,
      "peerIpAddress": null,
      "service": null,
      "destAddress": null,
      "peerLocation": null,
      "peerSiteReputation": null,
      "peerFqdn": null
    },
    "eventId": "depersonalized",
    "deviceDetails": {
      "deviceType": "MAC",
      "policyName": "depersonalized",
      "targetPriorityCode": "depersonalized",
      "msmGroupId": 0,
      "deviceVersion": "depersonalized",
      "msmGroupName": null,
      "deviceIpAddress": "depersonalized",
      "deviceId": "depersonalized",
      "deviceName": "depersonalized",
      "agentLocation": "depersonalized",
      "loginUserName": null,
      "policyId": "depersonalized",
      "deviceOwnerName": null,
      "email": "depersonalized",
      "deviceHostName": null,
      "targetPriorityType": "depersonalized",
      "deviceIpV4Address": "depersonalized",
      "deviceLocation": {
        "areaCode": 0,
        "dmaCode": 0,
        "city": null,
        "countryCode": "depersonalized",
        "latitude": "depersonalized",
        "metroCode": 0,
        "postalCode": null,
        "countryName": "depersonalized",
        "region": null,
        "longitude": "-depersonalized"
      }
    },
    "selectedApp": {
      "reputationProperty": "TRUSTED_WHITE_LIST",
      "applicationPath": "/bin/zsh",
      "md5Hash": "78ece0c98881f7e2052fb04186e9a46d",
      "sha256Hash": "1f41e79753e4a75dfee7a9bd309a40ed21e7b69523e8b152e3df4ad71455d8f2",
      "virusSubCategory": null,
      "virusName": null,
      "effectiveReputationSource": "CLOUD",
      "effectiveReputation": "TRUSTED_WHITE_LIST",
      "applicationName": "zsh",
      "virusCategory": null
    },
    "threatIndicators": [
      "RUN_CMD_SHELL",
      "MITRE_T1059_CMD_LINE_OR_SCRIPT_INTER"
    ],
    "shortDescription": "The application \"<share><link hash=\"1f41e79753e4a75dfee7a9bd309a40ed21e7b69523e8b152e3df4ad71455d8f2\">zsh</link></share>\" invoked the application \"<share><link hash=\"b0ddc6174508b39afd43810089d8e00ba423a714464a4aa9fcbb40480d40bec0\">bash</link></share>\".",
    "eventType": "CREATE_PROCESS",
    "securityEventCode": null,
    "parentApp": {
      "reputationProperty": "TRUSTED_WHITE_LIST",
      "applicationPath": null,
      "md5Hash": null,
      "sha256Hash": "0e0bcb6055164fa9ccebcab0fa4a9f8230afc5c76746e09064ff0f4512427075",
      "virusSubCategory": null,
      "virusName": null,
      "effectiveReputationSource": "CLOUD",
      "effectiveReputation": "TRUSTED_WHITE_LIST",
      "applicationName": "login",
      "virusCategory": null
    },
    "targetApp": {
      "reputationProperty": "ADAPTIVE_WHITE_LIST",
      "applicationPath": null,
      "md5Hash": null,
      "sha256Hash": "b0ddc6174508b39afd43810089d8e00ba423a714464a4aa9fcbb40480d40bec0",
      "virusSubCategory": null,
      "virusName": null,
      "effectiveReputationSource": "VECTOR",
      "effectiveReputation": "LOCAL_WHITE",
      "applicationName": "bash",
      "virusCategory": null
    },
    "@timestamp": "2020-10-26T21:38:00.000000000+00:00",
    "createTime": 1603748316442,
    "registryValue": null,
    "attackStage": null,
    "incidentId": null
  },
  "fields": {
    "@timestamp": [
      "2020-10-26T21:38:00.000Z"
    ]
  }
}
```

This way, if you will use the `process_creation` category, it should work for it as well (as long as there is a common data model in place). 

The `product ` is mostly about an operating system, not about a logging subsystem/sensor, etc.
The categories have been implemented to simplify these things (:

Reply:
@snake-jump yes, there is an open issue around folder structure that I opened in here https://github.com/Neo23x0/sigma/issues/1048 if you want to add some of your comments, they make total sense.